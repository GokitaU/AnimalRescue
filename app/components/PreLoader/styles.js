import { StyleSheet, Dimensions } from "react-native";
import { COLOR_PRIMARY, COLOR_GRAY } from "../../config/styles";

let deviceWidth = Dimensions.get("window").width;
let deviceHeight = Dimensions.get("window").height;

export default StyleSheet.create({
    container: {
        flex: 1,
        backgroundColor: "#192f6a",
        justifyContent: "center",
        alignItems: "center"
    },
    companyName: {
        fontSize: 32,
        fontWeight: "600",
        color: "#FFFFFF",
    },
    slogan: {
        fontSize: 18,
        fontWeight: "600",
        color: "#228B22",
        marginTop: 10,
    },
    image: {
        marginTop: "40%",
        width: 120,
        height: 150,
        alignSelf: "center"
    }
});