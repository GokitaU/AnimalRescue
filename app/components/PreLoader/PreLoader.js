import React, { Component } from "react";
import { Text, View, Image } from "react-native";
import styles from "./styles";
import { WaveIndicator } from "react-native-indicators";
export default class PreLoader extends Component {
    render() {
        return (
            <View style={styles.container}>
                <Image source={require("../../images/ara.png")} style={styles.image} />
                <Text style={styles.companyName}>Animal Rescue App</Text>
                <Text style={styles.slogan}>All life is important, no matter how small.</Text>
                <WaveIndicator size={100} color={"#fff"} />
            </View>
        );
    }
}