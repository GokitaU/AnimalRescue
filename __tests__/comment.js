import React from 'react';
import PreLoader from '../app/components/PreLoader/PreLoader';
import 'react-native';
import renderer from 'react-test-renderer';

jest.useFakeTimers();

test('renders correctly', async () => {
    const tree = await renderer.create(<PreLoader />).toJSON();
    expect(tree).toMatchSnapshot();
});





