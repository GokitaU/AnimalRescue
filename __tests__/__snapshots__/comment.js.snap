// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#192f6a",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Image
    source={
      Object {
        "testUri": "../../../app/images/ara.png",
      }
    }
    style={
      Object {
        "alignSelf": "center",
        "height": 150,
        "marginTop": "40%",
        "width": 120,
      }
    }
  />
  <Text
    style={
      Object {
        "color": "#FFFFFF",
        "fontSize": 32,
        "fontWeight": "600",
      }
    }
  >
    Animal Rescue App
  </Text>
  <Text
    style={
      Object {
        "color": "#228B22",
        "fontSize": 18,
        "fontWeight": "600",
        "marginTop": 10,
      }
    }
  >
    All life is important, no matter how small.
  </Text>
  <View
    style={
      Array [
        Object {
          "alignItems": "center",
          "flex": 1,
          "justifyContent": "center",
        },
        undefined,
      ]
    }
  >
    <View
      animating={true}
      animationDuration={1600}
      animationEasing={[Function]}
      color="#fff"
      interaction={true}
      renderComponent={[Function]}
      style={
        Object {
          "height": 100,
          "width": 100,
        }
      }
      waveFactor={0.54}
      waveMode="fill"
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "bottom": 0,
            "justifyContent": "center",
            "left": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "#fff",
              "borderRadius": 50,
              "borderWidth": 0,
              "height": 100,
              "opacity": 1,
              "transform": Array [
                Object {
                  "scale": 0,
                },
              ],
              "width": 100,
            }
          }
        />
      </View>
      <View
        style={
          Object {
            "alignItems": "center",
            "bottom": 0,
            "justifyContent": "center",
            "left": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "#fff",
              "borderRadius": 50,
              "borderWidth": 0,
              "height": 100,
              "opacity": 1,
              "transform": Array [
                Object {
                  "scale": 0,
                },
              ],
              "width": 100,
            }
          }
        />
      </View>
      <View
        style={
          Object {
            "alignItems": "center",
            "bottom": 0,
            "justifyContent": "center",
            "left": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "#fff",
              "borderRadius": 50,
              "borderWidth": 0,
              "height": 100,
              "opacity": 1,
              "transform": Array [
                Object {
                  "scale": 0,
                },
              ],
              "width": 100,
            }
          }
        />
      </View>
      <View
        style={
          Object {
            "alignItems": "center",
            "bottom": 0,
            "justifyContent": "center",
            "left": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "#fff",
              "borderRadius": 50,
              "borderWidth": 0,
              "height": 100,
              "opacity": 1,
              "transform": Array [
                Object {
                  "scale": 0,
                },
              ],
              "width": 100,
            }
          }
        />
      </View>
    </View>
  </View>
</View>
`;
